packages/components/src/internal/components/samples/ManageSampleStatusesPanel.tsx

```diff
@@ -284,39 +308,71 @@ export const SampleStatusDetail: FC<SampleStatusDetailProps> = memo(props => {
 SampleStatusDetail.displayName = 'SampleStatusDetail';
 
 interface SampleStatusesListProps {
-    onSelect: (index: number) => void;
+    onSelect: (index: number, group?: string) => void;
     selected: number;
-    states: SampleState[];
+    selectedGroup: string;
+    statesByType: Record<string, SampleState[]>;
 }
 
+const HELP_TEXT = {
+    [SampleStateType.Available]: 'Allows any action. Included in the calculation of available amount.',
+    [SampleStateType.Consumed]: 'Prevents updates to storage such as adding to storage or changing F/T count.',
+    [SampleStateType.Locked]: 'All updates to the sample are prevented. Locked samples may be added to picklists.',
+};
+
 // exported for jest testing
 export const SampleStatusesList: FC<SampleStatusesListProps> = memo(props => {
-    const { states, onSelect, selected } = props;
+    const { statesByType, onSelect, selected, selectedGroup } = props;
 
     return (
-        <div className="list-group">
-            {states.map((state, index) => (
-                <ChoicesListItem
-                    active={index === selected}
-                    index={index}
-                    subLabel={state.stateType !== state.label && state.stateType}
-                    key={state.rowId}
-                    label={state.label}
-                    onSelect={onSelect}
-                    componentRight={
-                        (state.inUse || !state.isLocal) && (
-                            <LockIcon
-                                iconCls="pull-right choices-list__locked"
-                                body={getSampleStatusLockedMessage(state, false)}
-                                id="sample-state-lock-icon"
-                                title={SAMPLE_STATUS_LOCKED_TITLE}
-                            />
-                        )
-                    }
-                />
+        <>
+            {Object.keys(statesByType)
+                .sort()
+                .map(stateType => (
+                    <React.Fragment key={stateType}>
+                        <div className="choice-section-header">
+                            {stateType}
+                            <LabelHelpTip
+                                iconComponent={
+                                    <span>
+                                        <i className="fa fa-info-circle" />
```

Comment:
isn't this info/help tip icon the default for LabelHelpTip when no iconComponent isn't provided?

Reply:
Ah, so it is. Thanks.